portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,10 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function callServlets() {
+  const responseFromServer = await fetch(/random-number');
+  const textFromResponse = await responseFromServer.text();
+  const dateContainer = document.getElementById('random-number-container');
```

Comment:
rename dateContainer to randomNumberContainer?

Reply:
yes!

Reply:
I had to chenge the name in index.html and script.js to work

Reply:
change*

Reply:
Oh, I mean should the var dateContainer here be changed to randomNumberContainer like:

`const randomNumberContainer = document.getElement...`

(not just the HTML element ID)